Source/UI/Packages/View Controllers/SwipeActionController.m

```diff
@@ -0,0 +1,69 @@
+//
+//  SwipeActionController.m
+//  Limitless
+//
+//  Created by Thatchapon Unprasert on 12/11/16.
+//  Copyright © 2016 Limitless. All rights reserved.
+//
+
+#import "SwipeActionController.h"
+
+@implementation SwipeActionController
```

Comment:
This class should probably be implemented via a sharedInstance with `BOOL` properties instead of `bool` since the former is an Objective-C type and the latter is C++'s. This would come with the usage of `YES` and `NO` instead of `true` and `false`.